README.md

```diff
@@ -37,30 +35,41 @@ In `/etc/hosts` add this:
 ```
 
 ### Setup Database
+- Follow the steps [here](https://mariadb.com/kb/en/installing-and-using-mariadb-via-docker/) to install mariadb via docker.
+- Then run the docker container using the following command.
+```bash
+docker run --detach --name mariadb-tumlive --env MARIADB_USER=root --env MARIADB_PASSWORD=root --env MARIADB_ROOT_PASSWORD=example --restart always -p 3306:3306 mariadb:latest`
```

Comment:
Is `--env MARIADB_PASSWORD=root` required?

Reply:
It is not required indeed, I found the list of available variables [here](https://mariadb.com/kb/en/mariadb-docker-environment-variables/)

Reply:
Removed it!